core/deployment/src/test/java/io/quarkiverse/operatorsdk/test/sources/SimpleReconciler.java

```diff
@@ -4,12 +4,14 @@
 import io.javaoperatorsdk.operator.api.reconciler.ControllerConfiguration;
 import io.javaoperatorsdk.operator.api.reconciler.Reconciler;
 import io.javaoperatorsdk.operator.api.reconciler.UpdateControl;
+import io.quarkiverse.operatorsdk.annotations.RBACCRoleRef;
 import io.quarkiverse.operatorsdk.annotations.RBACRule;
 import io.quarkiverse.operatorsdk.annotations.RBACVerbs;
 
 @ControllerConfiguration(name = SimpleReconciler.NAME)
 @RBACRule(verbs = RBACVerbs.UPDATE, apiGroups = SimpleReconciler.CERTIFICATES_K8S_IO_GROUP, resources = SimpleReconciler.ADDITIONAL_UPDATE_RESOURCE)
 @RBACRule(verbs = SimpleReconciler.SIGNERS_VERB, apiGroups = SimpleReconciler.CERTIFICATES_K8S_IO_GROUP, resources = SimpleReconciler.SIGNERS_RESOURCE, resourceNames = SimpleReconciler.SIGNERS_RESOURCE_NAMES)
+@RBACCRoleRef(name = SimpleReconciler.ROLE_REF_NAME, apiGroup = SimpleReconciler.RBAC_AUTHORIZATION_GROUP, kind = SimpleReconciler.ROLE_REF_KIND)
```

Comment:
Could you add a test that checks that a matching cluster role binding is indeed added to the generated kubernetes manifests, please?

Reply:
done